background.ts

```diff
@@ -1228,6 +1230,22 @@ class BackgroundCommandWorker implements CommandWorkerInterface {
       }
     }
   }
+
+  async listSnapshots(context: CommandWorkerInterface.CommandContext): Promise<Snapshot[]> {
```

Comment:
FWIW, you can leave off the return types here (it'll get inferred from the implementation), which lets us drop off an import.